installer/conf/rsyslog-lad.conf

```diff
@@ -0,0 +1,2 @@
+# OMS Syslog collection for workspace LAD (SYSLOG_PORT=%SYSLOG_PORT%)
```

Comment:
nit: remove "OMS" from the descriptive text, since OMS service really isn't involved.